www/source/index.html.slim

```diff
@@ -14,28 +14,21 @@ header.blue-gradient.margin-top-offset
    canvas.particles-js-canvas-el /
   /! style for callout icons
   .row
-    .icon.shadow.code-trigger
-      img src="/images/open-close.svg" /
+    .icon.shadow.hide-for-small-only
+      img src="/images/circ-arrow.svg" /
       
 /! Surprise Code example
 .code-pop
   .strict-center
-    p.code.code-demo.code-pop-animate
-      span.code-attribute
-        | describe 
-      | code
-      span.code-attribute
-        |  do it
-      |  { should be_inspec }
-      span.code-attribute
-        |  end
-
-      
+    p.t-white
+      | Upcoming Webinar: Testing at the Edges
+    a href="https://events.chef.io/events/testing-at-the-edges/"  View the Event
```

Comment:
I think this event on March 1 and has come-and-gone... we probably need to put a new event here or perhaps put the surprise code example back.